doc/related_projects.rst

```diff
@@ -285,7 +285,7 @@ Other packages useful for data analysis and machine learning.
   statistical models. More focused on statistical tests and less on prediction
   than scikit-learn.
 
-- `PyMC <https://pymc-devs.github.io/pymc/>`_ Bayesian statistical models and
+- `PyMC <https://github.com/pymc-devs/pymc>`_ Bayesian statistical models and
```

Comment:
we probably should point to the docs as before: `https://www.pymc.io/`

Reply:
Thanks @adrinjalali, I applied your suggestion.